### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by chrismaster** [Santulator/Santulator#11]
Some kind of auto update with eg update4j would be cool.

**Comment by AdamCarroll** [comment]
Yes, I think a feature like that would be useful.  I'll update this issue and leave it here in the backlog.
